src/pages/Home/Home.scss

```diff
@@ -12,7 +12,8 @@
   }
 
   &__description {
-    text-align: justify;
+    display: block;
```

Comment:
No need to have a `display: block` if the html element is already a block element (div, p)